include/filters.php

```diff
@@ -376,7 +383,10 @@ public function translate_page_for_privacy_policy( $id ) {
 	 * @return array
 	 */
 	public function fix_privacy_policy_page_editing( $caps, $cap, $user_id, $args ) {
-		if ( in_array( $cap, array( 'edit_page', 'edit_post', 'delete_page', 'delete_post' ) ) ) {
+		static $once = true; // Avoid infinite loop.
```

Comment:
1. precise the context for the infinite loop. I suggest:
```
// Avoid infinite loop if a capability check is made in a method hooked to `get_translations()`.
```
2. I fear you are introducing new bugs in case we check capabilities for a different user or a different page after the first call.